lib/v2/zyw/radar.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    'zyw.asia': {
+        _name: 'zyw',
+        hot: [
+            {
+                title: '今日热榜',
+                docs: 'https://docs.rsshub.app/new-media.html#zyw-jin-ri-re-bang',
+                source: ['/#/list', '/'],
```

Comment:
```suggestion
                source: ['/'],
```
https://github.com/DIYgod/RSSHub/actions/runs/5015935242/jobs/8992161258?pr=12527#step:6:16